src/style.css

```diff
@@ -0,0 +1,119 @@
+:root {
+  --primary-color: #3d4e61;
+  --secondery-color: #e1e7ed;
+  --shadow-color: #3d4e611a;
+  --background-color: #fff;
+  --icon-color: #687c94;
+  --box-shadow-color-one: #00000040;
+  --box-shadow-color-two: #3d4e6133;
+}
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf")
+    format("truetype");
+  font-weight: 300;
+  letter-spacing: 0;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf")
+    format("truetype");
+  font-weight: 300;
+  letter-spacing: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  position: absolute;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: calc(100% - 18px);
+  box-shadow: 0 1px 8px 0 var(--shadow-color);
+  border-radius: 3px;
+  border: 1px solid var(--secondery-color);
+  color: var(----primary-color);
+}
+
+.search-bar__big {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 20px 23px;
+  color: var(--icon-color);
+  height: 68px;
+  background-color: var(--background-color);
+}
+
+.search-bar__small {
+  background-image: url("images/Icon.svg");
+  background-repeat: no-repeat;
+  background-position: 12px 14px;
+  color: var(--icon-color);
+  margin-top: 20px;
+  height: 40px;
+  border: 1px solid var(--secondery-color);
+  margin-bottom: 10px;
+}
+
+.input {
+  height: 19px;
+  line-height: 19px;
+  width: 158px;
+  text-align: left;
+  margin-bottom: 2px;
+  color: var(--primary-color);
+  font-family: "Avenir-Book",
+    sans-serif;
+  border-color: transparent;
+}
+
+.big__input {
+  margin-left: 60px;
+  font-size: 16px;
+  margin-bottom: 2px;
+}
+
+.small__input {
+  top: 25px;
+  left: 62px;
+  margin-left: 31px;
+  font-size: 14px;
+}
+
+.big__input:hover {
+  border-color: var(--secondery-color);
+  box-shadow: 0 4px 4px 0 var(--box-shadow-color-one);
+}
+
+.big__input:focus {
+  border: 1px solid var(--secondery-color);
+}
+
+.small__input:hover {
+  border: 1px solid var(--secondery-color);
+  border-color: var(--secondery-color);
+  box-shadow: 0 3px 8px 0 var(--box-shadow-color-two);
+}
+
+.small__input:focus {
+  font-family: "Avenir-Heavy",
+    sans-serif;
```

Comment:
```suggestion
  font-family: "Avenir-Heavy", sans-serif;
```

Reply:
Before, there was absolutely opposite advice above.

Reply:
it was about html code, not css